network_diffusion/mln/functions.py

```diff
@@ -26,6 +26,31 @@
 from network_diffusion.mln.mlnetwork import MultilayerNetwork
 
 
+def betweennes(net: MultilayerNetwork) -> Dict[MLNetworkActor, int]:
+    """Return value of centrality for actors from the network"""
+    bet: Dict[MLNetworkActor, int] = {}
```

Comment:
these two functions return a dict keyed by MLNactor with values of betweenness/closeness in a rendomly chosen layer! This approach will not work